code/modules/client/preferences/prisoner_crime.dm

```diff
@@ -0,0 +1,20 @@
+/// Which crime is the prisoner permabrigged for. For fluff!
+/datum/preference/choiced/prisoner_crime
+	category = PREFERENCE_CATEGORY_NON_CONTEXTUAL
+	savefile_identifier = PREFERENCE_CHARACTER
+	savefile_key = "prisoner_crime"
+
+/datum/preference/choiced/prisoner_crime/init_possible_values()
+	return GLOB.prisoner_crimes
+
+/datum/preference/choiced/prisoner_crime/apply_to_human(mob/living/carbon/human/target, value)
+	return
+
+/datum/preference/choiced/prisoner_crime/create_default_value()
+	return pick(GLOB.prisoner_crimes)
+
+/datum/preference/choiced/prisoner_crime/is_accessible(datum/preferences/preferences)
```

Comment:
This should probably check if your highest job is prisoner, just like AI core display: https://github.com/tgstation/tgstation/blob/d91390a4474e511279b6ee7525c5982266b9a858/code/modules/client/preferences/ai_core_display.dm#L18-L22